### PR TITLE
Add helper messages for Minecraft Legends

### PR DESCRIPTION
**Pull request by violine1101**
Mostly just copied the messages for dungeons

### CODE REVIEW

File: assets/messages.yml [hunk 1]
```diff
@@ -876,6 +900,19 @@ messages:
...
+    - project:
+        - mclg
+      name: Minimum requirements not met
+      shortcut: min
+      category: invalid
+      message: |-
+        *Thank you for your report!* 
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        Your computer does not meet the [minimum system requirements|https://help.minecraft.net/hc/en-us/articles/360038937032-Dungeons-Minimum-Specifications-for-Gameplay] to run Minecraft Legends.
```
Comment (osfanbuff63) [inline assets/messages.yml:914]: I'm assuming there's no Legends-specific one?

**Review by osfanbuff63 — APPROVED** [review]
Seems good to me

File: assets/messages.yml (hunk 1 above)
Comment (violine1101) [reply assets/messages.yml:914]: Currently trying to find that out, at the very least I haven't found anything official.

Comment (violine1101) [reply assets/messages.yml:914]: I'll leave this as is for now, we can always change it later.

**Pull request merged**